rgeomstats/manifold.R

```diff
@@ -0,0 +1,16 @@
+# Manifold, i.e. a topological space that locally resembles
+# Euclidean space near each point.
+
+library(reticulate)
+gs <- import_from_path("geomstats", path = ".")
+
+Manifold <- setRefClass("Manifold",
+                        fields = c("dimension"),
```

Comment:
why is it `c("dimension")` here and only `"dimension"` for the Euclidean space? Could you uniformize? thanks!